### ISSUE TITLE
[BUG](2.3) Async is not working on ts-framework console

### ISSUE THREAD

**Issue by jmbrito01** [nxtep-io/ts-framework#26]
Tried adding a new "Domain" from ts-framework-sql models and it's returning this:
```bash
bitcapital-core > await Domain.create({ name: "DOmain test", role: "not-working", postbakcUrl: 'https://'}).save()
await Domain.create({ name: "DOmain test", role: "not-working", postbakcUrl: 'https://'}).save()
^^^^^

SyntaxError: await is only valid in async function
```

**Comment by jmbrito01** [comment]
Tried adding a new "Domain" from ts-framework-sql models and it's returning this:
```bash
bitcapital-core > await Domain.create({ name: "DOmain test", role: "not-working", postbakcUrl: 'https://'}).save()
await Domain.create({ name: "DOmain test", role: "not-working", postbakcUrl: 'https://'}).save()
^^^^^

SyntaxError: await is only valid in async function
```

**Comment by luiseduardobrito** [comment]
Fixed in #23
